README.md

```diff
@@ -28,9 +28,9 @@ This project is still under development. The following features with the check m
 - Rust >= 1.19.0
 - binutils >= 2.22
 - LLVM and Clang >= 3.9 recommended by bindgen
-- By default, the [secure feature](#feature-secure) is enabled, therefore Go (>=1.7) is required.
+- By default, the [boringssl](#feature-secure) is enabled, therefore Go (>=1.7) is required.
```

Comment:
I don't think there is such a features named boringssl. Better express in another way.

Reply:
secure bring misunderstanding, because only boringssl depends on go. maybe change to the following：

the [secure feature](#feature-secure) is enabled and implemented by boringssl

Reply:
How about
```
By default, the secure features is provided by boringssl, which requires Go to build. You can also use openssl instead by enable features openssl.
```
?

@nrc what do you think?

Reply:
sgtm